### ISSUE TITLE
Issue parsing Last-Modifed value with Time.httpdate

### ISSUE THREAD

**Issue by ianks** [boazsegev/iodine#64]
Hey @boazsegev,

We are getting some errors when attempting to parse Last-Modified headers, and I think the error may be coming from iodine generated timestamps. Ruby's `httpdate` fails when parsing dates without two digits for the day number. I believe this may actually be a ruby bug (couldn't really tell from the spec) but it would be hugely helpful if iodine could bypass this by ensuring two digits :)

### Does not work
```
> Time.httpdate('Thu, 6 Mar 2019 05:17:22 GMT')
ArgumentError: not RFC 2616 compliant date: "Thu, 6 Mar 2019 05:17:22 GMT"
```

### Works

```
> Time.httpdate('Thu, 06 Mar 2019 05:17:22 GMT')
ArgumentError: not RFC 2616 compliant date: "Thu, 6 Mar 2019 05:17:22 GMT"
```

Thank you so much!

**Comment by boazsegev** [comment]
Thank you for opening this issue! 🙏🏻👍🏻☺️

I'm authoring a fix. I'll try to release the patch today or tomorrow.

Kindly,
   Bo.

**Comment by boazsegev** [comment]
I released the patch as version 0.7.26.

Thanks again for exposing the issue!
🙏🏻

**Comment by ianks** [comment]
You're the best!
